source/htslib/package.d

```diff
@@ -0,0 +1,24 @@
+module htslib;
```

Comment:
this is a good idea, do it in a different pull request because it is independent of this change..